lib/cluster.js

```diff
@@ -523,6 +528,35 @@ function workerInit() {
       process: process,
       state: 'online'
     });
+
+    worker._disconnect = function(masterInitiated) {
```

Comment:
This will still be exposed to the end user. Can we make it private to this module rather than exposing it or is there a reason it cannot be done that way?